src/Microsoft.AspNetCore.Mvc.Razor.Host/ViewComponentTagHelperPass.cs

```diff
@@ -11,21 +11,21 @@
 
 namespace Microsoft.AspNetCore.Mvc.Razor.Host
 {
-    public class ViewComponentTagHelperPass : IRazorIRPass
+    public class ViewComponentTagHelperPass : IRazorIROptimizationPass
     {
         public RazorEngine Engine { get; set; }
 
-        public int Order => RazorIRPass.LoweringOrder;
+        public int Order => RazorIRPass.DefaultFeatureOrder;
```

Comment:
I think having `DefaultFeatureOrder` be a higher value than 0 is useful especially in cases like `DefaultDocumentClassifierPass`. Mvc and RazorPages uses their own classifiers and they shouldn't have to use negative numbers to override the Order.